docker_images/sklearn/Dockerfile

```diff
@@ -1,29 +1,40 @@
-FROM tiangolo/uvicorn-gunicorn:python3.8
-LABEL maintainer="Omar Sanseviero omar@huggingface.co"
+FROM mambaorg/micromamba
```

Comment:
Are there any Python requirements we could pre-install to make the loading as fast as possible? (i know sklearn is dynamic, but more generally I was wondering about other deps)

Reply:
we're kinda not even fixing the python version here. For an older version of sklearn which doesn't support a new python release, we'll have the latest python that it supports, and for the new ones the latest python release. And once python itself becomes not fixed, I'm not sure what we can do about the rest lol

But we can do a local channel deployment at some point on our infra to reduce traffic and increase speed. A basic sklearn env at this point does about 140MB of download. We could also have a proxy and cache all of those.